spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe "GET /show " do
+    it "responds with 200" do
+      get :show
+      expect(response).to have_http_status(200)
+    end
+    it "adds user to database" do
+      expect{User.create}.to change{User.count}.by(1)
```

Comment:
Can we add spaces around the curly braces, as per the Ruby Style Guide?
i.e. `expect { User.create }.to change { User.count } `